docs/apis-tools/frontend-development/01-task-applications/02-user-task-lifecycle.md

```diff
@@ -0,0 +1,148 @@
+        D -->|resume| B(In progress)
+        B -->|return| A
+        style New fill:black
+        style C stroke-width:2px
+    end
+```
+
+:::tip
+To benefit from [out-of-the-box task performance reporting in Optimize](#task-lifecycle-reporting-in-optimize), your task lifecycle should cover the outlined work state actions `start` and `complete`, and optionally `pause`, `resume`, and `return`.
+:::
+
+### Task assignment
+
+The assignment logic runs parallel to the work state. This ensures maximum flexibility. For example, a task may be assigned but be open for a while (lying in the inbox), indicating that the assigned user is not available to work on this task promptly and offering optimization potential for assignment rules for this task. Another example is the change of task assignee while working on a task, such as in collaborative environments.
+
+In the Camunda Tasklist web application, a task can be claimed by the logged-in user, which assigns the task to that user. Managers can assign unassigned tasks to team members and reassign them as needed.
+
+```mermaid
+flowchart
+    subgraph Assignment
+        Unassigned(Unassigned) -->|assign/claim| Assigned(fa:fa-user Assigned)
+        Assigned -->|return| Unassigned
+        Assigned -->|reassign| Assigned
+    end
+```
+
+The execution engine does not validate whether a user is authorized to execute a task; instead, this is a matter for the application that is built on top. For example, the Camunda Tasklist web application checks that only the assigned user or an admin/manager is able to update and complete a task, but you are free to use different logic in your task application. This way, maximum flexibility is guaranteed when you create a custom application based on Camunda. For example, you can implement a substitute logic that allows a user to complete a task on behalf of another user.
+
+Our best practices, as implemented in the Tasklist web application, are as follows:
+
+- `update` and `complete` operations can only be performed by the assigned user or an admin/manager.
+- Users can only see tasks to which they are assigned and tasks to which a user group of which they are a member is assigned as a candidate.
+- When a task is returned to the queue (i.e. the assignee is cleared), its data and status are reset to "open".
+- Only admins/managers can reassign tasks.
+- Task agents can return tasks, but must provide a comment as to why they are doing so.
+- Task agents can mark tasks with a follow-up date. These then disappear from their individual task list until the follow-up date is reached. The `open` status is preserved, or the task is moved to the `paused` status if it has already been processed. The task remains assigned to the user.
+
+The following image demonstrates how this is similarly implemented in the Camunda Tasklist web application:
+
+<img src={TaskLifecycleUI} style={{width: 600}} alt="Task lifecycle and assignment in the Camunda Tasklist UI" />
```

Comment:
Do you prefer to outline your images in this format for a particular reason, rather than ![]()? Just curious if you don't feel the default Markdown layout doesn't meet the formatting you would prefer.

Reply:
Hi @christinaausley , using Markdown, I can't define the width, and images will scale too much and look distorted. Especially when I just took a screenshot of a small part of the screen, it would be enlarged heavily and hard to read